jenkins/job-configs/kubernetes-jenkins/kubernetes-garbagecollector.yaml

```diff
@@ -72,5 +72,15 @@
                 export KUBEMARK_NUM_NODES="100"
                 # The kubemark scripts build a Docker image
                 export JENKINS_ENABLE_DOCKER_IN_DOCKER="y"
+        - 'e2e-gce':  # kubernetes-garbagecollector-e2e-gce
+            cron-string: 'H H/6 * * *'
+            description: 'Runs all non-slow, non-serial, non-flaky, tests on GCE in parallel, with the Garbage Collector turned on'
+            timeout: 50
+            job-env: |
```

Comment:
ah, I see you already have that for -feature:
`export FAIL_ON_GCP_RESOURCE_LEAK="false"`